markdownfmt/markdownfmt.go

```diff
@@ -27,9 +27,12 @@ func NewGoldmark(opts ...renderer.Option) goldmark.Markdown {
 	gm := goldmark.New(
 		goldmark.WithExtensions(extensions...),
 		goldmark.WithParserOptions(parserOptions...),
+		goldmark.WithRenderer(mr), // needs to be before WithRendererOptions
+		// not sure if we need to use goldmark's WithRendererOptions that... just forwards it to our AddOptions
+		// but, see AddOptions comment in markdown/renderer.go
```

Comment:
Make sense. If we want this comment let's:

* Not mix line and block comment. Ideally, we have this comment above if multiline
* Let's use full sentences (readability) 🤗 
* 
```suggestion
		// Put our renderer before WithRendererOptions, so we can leverage them to configure renderer (Goldmark pattern).
		goldmark.WithRenderer(mr),
```